### PR TITLE
Subscription options fixes

### LINKED ISSUE #10: SubscriptionOptions.setStartAtSequence does not return SubscriptionOptions (this)
from index.d.ts
```javascript
setStartAtSequence(sequence: number): SubscriptionOptions;`
```
from stan.js
```javascript
SubscriptionOptions.prototype.setStartAtSequence = function(sequence) {
  this.startPosition = proto.StartPosition.SEQUENCE_START;
  this.startSequence = sequence;
};
```

### PR DESCRIPTION
**Pull request by aricart**
3 embarrassing bugs:
 - Missing return from subscriptionOptions#setStartAtSequence didn't return itself to allow chaining.
 - Calls to `date.millis()` instead of `date.getTime()`

Resolves #10

### CODE REVIEW

**Comment by aricart** [comment]
A pair of eyes welcomed. @kozlovic @ColinSullivan1

**Comment by kozlovic** [comment]
LGTM

File: lib/stan.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ var util = require('util'),
 /**
  * Constants
  */
-var VERSION = '0.0.3',
+var VERSION = '0.0.4',
```
Comment (sfrooster) [inline lib/stan.js]: npm publish on version bump (at least)?

Comment (aricart) [inline lib/stan.js]: It is published. You can also reference from npm the github repo to get the current master.

**Comment by sfrooster** [comment]
It doesn't look like it is:

$ npm list node-nats-streaming
ts20@1.0.0 /Users/mhaan/AppDev/ts20
└── node-nats-streaming@0.0.3

I know I can pull the repo down, but it's not crucial so I'll wait for the
update.

**Comment by aricart** [comment]
Looks like they have not yet taken it.

> ~/go/src/git...-nats-streaming]$ npm publish https://github.com/nats-io/node-nats-streaming.git -tag node-nats-streaming@0.0.4 --access public
> ....
> npm ERR! "You cannot publish over the previously published version 0.0.4." : node-nats-streaming

**Comment by sfrooster** [comment]
Hmmm, ok.  No worries - thanks for following up!

**Comment by aricart** [comment]
@sfrooster the npm build is updated. Something was stuck there, but it is now fixed.

**Pull request merged**